.github/workflows/main.yml

```diff
@@ -112,7 +116,57 @@ jobs:
           # We ignore it when no files could be found because the test command
           # could also be failed because of an unit test. In this case there
           # would be no failed golden test to upload.
-          if-no-files-found: ignore   
+          if-no-files-found: ignore
+  
+  android-integration-test:
+    # Is recommended to run the job with a macOS machine to take advantage of
+    # hardware acceleration support provided by HAXM. See more details in the
+    # README.md of the Android emulator action:
+    # https://github.com/ReactiveCircus/android-emulator-runner#usage
+    runs-on: macos-latest
+    if: ${{ github.event.pull_request.draft == false }}
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
For what is this exactly needed again? Couldn't find anything mentioning this step at https://github.com/ReactiveCircus/android-emulator-runner

Reply:
Can you add this via a comment above it?

Reply:
done